db/migrate/20230815173748_create_comments.rb

```diff
@@ -0,0 +1,11 @@
+class CreateComments < ActiveRecord::Migration[7.0]
+  def change
+    create_table :comments do |t|
+      t.text :content
```

Comment:
[REQUIRED] Kindly use text instead of content just like the ERD diagram

Reply:
I have changed the content to text.